autorevision.sh

```diff
@@ -1029,7 +1029,7 @@ pathSegment() {
 	local depth="0"
 
 	if [ ! -z "${pathz}" ]; then
-		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ]; do
+		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ] && [ ! "${pathz}" = "C:" ]; do
```

Comment:
It just hangs….  doing more investigation now… will send you details shortly




> On May 14, 2016, at 5:45 PM, dak180 <notifications@github.com> wrote:
> 
> In autorevision.sh <https://github.com/Autorevision/autorevision/pull/28#discussion_r63282564>:
> 
> > @@ -1029,7 +1029,7 @@ pathSegment() {
> >  	local depth="0"
> >  
> >  	if [ ! -z "${pathz}" ]; then
> > -		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ]; do
> > +		while [ ! "${pathz}" = "/" ] && [ ! "${pathz}" = "." ] && [ ! "${pathz}" = "C:" ]; do
> What happen if instead of [ ! "${pathz}" = "C:" ] you have [ ! "${pathz}" = "" ]?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/Autorevision/autorevision/pull/28/files/7f10a64d441613399883493c5fe5a3b49b9dc3c5#r63282564>